psi-notify.service

```diff
@@ -1,12 +1,17 @@
+[Unit]
+Description=notify on system-wide resource pressure using PSI
+Documentation=man:psi-notify(1)
+PartOf=graphical-session.target
+ConditionUser=!root
```

Comment:
psi-notify shouldn't be run as root, but I don't think we need to explicitly add a `ConditionUser` for it, unless there's some compelling reason?

By default there's no user session for root anyway.